src/utils/copy-to-clipboard.js

```diff
@@ -0,0 +1,19 @@
+const copyToClipboard = (text, callback) => {
+    if (typeof window !== 'undefined') {
```

Comment:
(nesting) please invert this if-statement and return early to avoid nesting and improve readability, like so: 

```js
const copyToClipboard = (text, callback) => {
  if (typeof window === 'undefined') { return; }
  const hiddenInput = document.createElement('input');
  hiddenInput.type = 'text';
  hiddenInput.style.height = '0px';
  hiddenInput.style.opacity = '0';
  hiddenInput.value = text;
  document.body.appendChild(hiddenInput);
  hiddenInput.select();
  document.execCommand('copy');
  document.body.removeChild(hiddenInput);
  callback();  
};
```